src/main/resources/javascript/myFilesDashboard.js

```diff
@@ -226,6 +145,9 @@ function bbAddFolder(rootFolderMissing) {
         }
     });
 }
+function editInContentEditor(uuid, locale) {
+    window.top.CE_API.edit(uuid, '', locale, locale);
```

Comment:
We probably need to test that window.top contains CE_API as content_editor is a module and can technically be deactivated

Reply:
Undefined check added)